README.md

```diff
@@ -1,52 +1,64 @@
-[![License](https://img.shields.io/github/license/ps3dev/ps3toolchain.svg)](./LICENSE)
+# PS3 Toolchain [![License](https://img.shields.io/github/license/ps3dev/ps3toolchain.svg)](./LICENSE)
 
-  What does this do?
- ====================
+## What does this do?
+- This program will automatically build and install a compiler and other
+tools used in the creation of homebrew software for the Sony PlayStation 3
+videogame system.
 
-  This program will automatically build and install a compiler and other
-  tools used in the creation of homebrew software for the Sony PlayStation 3
-  videogame system.
+## How do I use it?
 
-  How do I use it?
- ==================
+1. Set the `PS3DEV` and `PS1LIGHT` environment variables together with the executable/folder path (Linux folder `~/.bashrc` example)
 
- 1) Set up your environment by installing the following software:
+```bash
+ export PS3DEV=/usr/local/ps3dev
+ export PSL1GHT=$PS3DEV
 
-  autoconf, automake, bison, flex, gcc, libelf, make, makeinfo,
-  ncurses, patch, python, subversion, wget, zlib, libtool, python,
-  bzip2, gmp, pkg-config, g++, libssl-dev, clang
+ export PATH=$PATH:$PS3DEV/bin
+ export PATH=$PATH:$PS3DEV/ppu/bin
+ export PATH=$PATH:$PS3DEV/spu/bin
+```
 
-## Linux
+2. Create the `$PS3DEV` folder (Linux example)
 
-  Specifically on debian-based systems, the following command line should
-  be enough to install everything necessary:
+```bash
+sudo mkdir $PS3DEV
+```
+
+3. Give yourself permission to write files to the `$PS3DEV` folder (Linux example)
 
 ```bash
-  apt-get install autoconf automake bison flex gcc libelf-dev make \
-    texinfo libncurses5-dev patch python subversion wget zlib1g-dev \
-    libtool libtool-bin python-dev bzip2 libgmp3-dev pkg-config g++ libssl-dev clang
+sudo chown -R $USER:$USER $PS3DEV
 ```
 
-## macOS
+4. Make sure the new variables are set in the shell (Linux command example `source ~/.bashrc`)
+
+### Install dependencies
+
+#### Linux
 
-  On macOS systems, if you have [Homebrew](http://brew.sh) package manager, the following command line should
-  be enough to install everything necessary:
+- For Linux, these are the necessary packages:
 
 ```bash
-brew install autoconf automake openssl libelf ncurses zlib gmp wget pkg-config
+autoconf automake bison flex gcc libelf-dev make texinfo libncurses5-dev patch python subversion wget zlib1g-dev libtool libtool-bin python-dev bzip2 libgmp3-dev pkg-config g++ libssl-dev clang
 ```
 
- 2) Add the following to your login script:
+#### macOS
+
+- And for macOS, these are:
+
 ```bash
-  export PS3DEV=/usr/local/ps3dev
-  export PSL1GHT=$PS3DEV
+autoconf automake openssl libelf ncurses zlib gmp wget pkg-config
+```
 
-  export PATH=$PATH:$PS3DEV/bin
-  export PATH=$PATH:$PS3DEV/ppu/bin
-  export PATH=$PATH:$PS3DEV/spu/bin
+### Install toolchain
+
+3. Run the toolchain script:
```

Comment:
This should now be 5. due to the added steps above.  Or maybe 6. if you want to make the install dependencies step number 5 (it used to be 1., but now it appears to be unnumbered).